en/docs/reference/mediators/log-mediator.md

```diff
@@ -89,6 +109,21 @@ A sample log output:
 [2024-09-09 15:23:03,998]  INFO {LogMediator} - {api:StockQuoteAPI} Processing user details : {"firstName":"Johne", "lastName": "Doe"} with Purchase data : {"itemCode": 8987, "price": 45}
 ```
 
+### Logging message ID
+
+In this example, we define a message template using [Synapse Expressions]({{base_path}}/reference/synapse-properties/synapse-expressions).
```

Comment:
Let's explain how the message ID is getting printed. The description and the topic are not related.

Reply:
Fixed with 05b1980